opentech/static_src/src/app/src/containers/ReviewInformation.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import ReviewsBlock from '@components/ReviewsBlock'
+import { getSubmissionOfID } from '@selectors/submissions'
+
+const ReviewInformation = ({ submission }) => {
+    return <ReviewsBlock submission={submission} />
```

Comment:
Should be passing in the review objects here i think. The `ReviewBlock` doesn't really care about the submission, just the reviews.